### PR TITLE
user profile first commit

### PR DESCRIPTION
**Pull request by dawnaara**
@cbeck can you check out my code please? I keep getting errors on the server but not sure how to fix it.

### CODE REVIEW

**Comment by cbeck** [comment]
1) nice pull request :-)
2) What errors are you getting specifically? Can you paste the stack trace here or send me a message?

**Comment by dawnaara** [comment]
@cbeck I think checkpoint 4 is complete. I am seeing my profile page after I sign in. The only issue is I'm having trouble populating the show page with any information besides user email. @user.email returns the email but @user.name doesn't return the user name and instead returns an error. Any advice here?

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  def show
+  	@user = User.find(params[:id])
+  end
+
+  def new
```
Comment (cbeck) [inline app/controllers/users_controller.rb:6]: Not necessary.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  	@user = User.new
+  end
+
+  def create
```
Comment (cbeck) [inline app/controllers/users_controller.rb:10]: Also not necessary.

File: config/routes.rb [hunk 3]
```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
-  devise_for :users
+  get 'users/show'
```
Comment (cbeck) [inline config/routes.rb:2]: You can remove this because of line 5

**Comment by cbeck** [comment]
I don't actually see where you are trying to add a user name. Email is part of devise, but name is not. Did you add it to your form? Also, once you do add it, there is something you have to do with devise to allow it to be allowed through and saved. Check this out:

http://stackoverflow.com/questions/11904105/devise-and-strong-parameters

Hope this helps!

**Pull request merged**